README.md

```diff
@@ -69,6 +69,20 @@ To utilize the contracts and deploy to a local testnet, you can install the code
 forge install https://github.com/Uniswap/v4-core
 ```
 
+To run Foundry and Yarn tests:
```

Comment:
Please can you separate these into instructions for foundry and hardhat tests separately?
Hardhat needs `yarn install`,  `yarn compile`, `yarn test`
Forge needs `forge build`, `forge test`

So would be nice to list them separately :)

Reply:
Also - we have some of these instructions already in `CONTRIBUTING.md`. If you would like to update the instructions maybe it makes sense to do so in that file :)